contracts/finance/andromeda-rate-limiting-withdrawals/src/contract.rs

```diff
@@ -122,7 +123,14 @@ pub fn execute(
     }
 }
 
-pub fn handle_execute(ctx: ExecuteContext, msg: ExecuteMsg) -> Result<Response, ContractError> {
+pub fn handle_execute(mut ctx: ExecuteContext, msg: ExecuteMsg) -> Result<Response, ContractError> {
+    call_action(
```

Comment:
The `Response` object from this function should be used to ensure the economics contract is called correctly.